plugins/module_utils/system_access_users_utils.py

```diff
@@ -63,6 +63,43 @@ class OPNSenseCryptReturnError(Exception):
     """
 
 
+class OPNSensePasswordVerifyReturnError(Exception):
+    """
+    Exception raised when the return value of the instance is not what is expected
+    """
+
+
+def password_verify(existing_user_password: str, password: str) -> bool:
```

Comment:
Might be safer to type the password parameter as Optional and return `False` directly if it is None.

Reply:
Thanks for the suggestion, fixed in 84675d1.